providers/aws/fargate/cluster.go

```diff
@@ -276,7 +276,7 @@ func (c *Cluster) GetPod(namespace string, name string) (*Pod, error) {
 	tag := buildTaskDefinitionTag(c.name, namespace, name)
 	pod, ok := c.pods[tag]
 	if !ok {
-		return nil, strongerrors.NotFound(fmt.Errorf("pod %s/%s is not found", namespace, name))
+		return nil, errdefs.NotFoundf("pod %s/%s is not found", namespace, name)
```

Comment:
It looks like we use both `namespace/pod`, and `namespace-pod` in error messages. Any reason not to standardize on `namespace/pod`?